Malt/Shaders/Common/Mapping.glsl

```diff
@@ -4,7 +4,7 @@
 #include "Common.glsl"
 
 /* META GLOBAL
-    @meta: category=Vector;
+    @meta: category=Vector; internal=True;
```

Comment:
you set the entire file as internal. What about `matcap_uv` and 'hdri_uv`? do we not want to have those available as vector nodes? or was that an accidental side effect